### PR TITLE
Style and clarity edits and adds new default execution timeout

### PR DESCRIPTION
**Pull request by AllysonRosenthal**
Did the unit for execution timeout change from milliseconds to seconds? I see seconds in the interface, but the existing docs specified milliseconds.

### CODE REVIEW

**Comment by zackmckenna** [comment]
It’s still in milliseconds, every other value on the ui is seconds though so it was better to leave it that way.

**Review by zackmckenna — APPROVED** [review]

File: docs/serverless/endpoints/send-requests.md [hunk 1]
```diff
@@ -38,18 +40,20 @@ To be notified of completed jobs, a URL can be passed within the top level of th
...
-This default behavior ensures that resources aren't indefinitely consumed by jobs that are unable to complete.
 To customize the management of job lifecycles and resource consumption, the following policies can be configured:
 
-- **Execution Timeout**: Specifies the maximum duration a job can run before it's automatically terminated. This limit helps prevent jobs from running indefinitely and consuming resources unnecessarily.
-- **Low Priority**: When set, the job is marked as low priority. This option is ideal for non-time-sensitive tasks that don't require immediate execution. Low priority jobs won't trigger a scale-up of resources and only execute when higher priority jobs aren't pending. When the queue is empty, low priority jobs are executed.
+- **Execution Timeout**: Specifies the maximum duration that a job can run before it's automatically terminated. This limit helps prevent jobs from running indefinitely and consuming resources. The default value is 600 seconds (10 minutes). You can overwrite the default by specifying `executionTimeout` in the job input.
```
[the remark this answers is not in the document]
Comment (AllysonRosenthal) [reply docs/serverless/endpoints/send-requests.md:55]: Yes, that was part of my motivation for checking the unit :)

File: docs/serverless/endpoints/send-requests.md [hunk 2]
```diff
@@ -38,36 +40,45 @@ To be notified of completed jobs, a URL can be passed within the top level of th
...
-This default behavior ensures that resources aren't indefinitely consumed by jobs that are unable to complete.
 To customize the management of job lifecycles and resource consumption, the following policies can be configured:
 
-- **Execution Timeout**: Specifies the maximum duration a job can run before it's automatically terminated. This limit helps prevent jobs from running indefinitely and consuming resources unnecessarily.
-- **Low Priority**: When set, the job is marked as low priority. This option is ideal for non-time-sensitive tasks that don't require immediate execution. Low priority jobs won't trigger a scale-up of resources and only execute when higher priority jobs aren't pending. When the queue is empty, low priority jobs are executed.
+- **Execution Timeout**: Specifies the maximum duration that a job can run before it's automatically terminated. This limit helps prevent jobs from running indefinitely and consuming resources. The default value is 600000 milliseconds (10 minutes). You can overwrite the value for a request by specifying `executionTimeout` in the job input.
+
+:::note
```
Comment (AllysonRosenthal) [inline docs/serverless/endpoints/send-requests.md:57]: @zackmckenna @rachfop how's this for clarifying the UI behavior?

File: docs/serverless/endpoints/send-requests.md [hunk 3]
```diff
@@ -38,36 +40,45 @@ To be notified of completed jobs, a URL can be passed within the top level of th
...
+
+Changing the **Execution Timeout** value through the web UI sets the value for all requests to an endpoint. You can still overwrite the value for individual requests with `executionTimeout` in the job input.
```
Comment (AllysonRosenthal) [inline docs/serverless/endpoints/send-requests.md:59]: @zackmckenna @rachfop how's this for clarifying the UI behavior?

File: docs/serverless/endpoints/send-requests.md [hunk 4]
```diff
@@ -38,36 +40,45 @@ To be notified of completed jobs, a URL can be passed within the top level of th
...
-This default behavior ensures that resources aren't indefinitely consumed by jobs that are unable to complete.
 To customize the management of job lifecycles and resource consumption, the following policies can be configured:
 
-- **Execution Timeout**: Specifies the maximum duration a job can run before it's automatically terminated. This limit helps prevent jobs from running indefinitely and consuming resources unnecessarily.
-- **Low Priority**: When set, the job is marked as low priority. This option is ideal for non-time-sensitive tasks that don't require immediate execution. Low priority jobs won't trigger a scale-up of resources and only execute when higher priority jobs aren't pending. When the queue is empty, low priority jobs are executed.
+- **Execution Timeout**: Specifies the maximum duration that a job can run before it's automatically terminated. This limit helps prevent jobs from running indefinitely and consuming resources. The default value is 600000 milliseconds (10 minutes). You can overwrite the value for a request by specifying `executionTimeout` in the job input.
```
Comment (AllysonRosenthal) [inline docs/serverless/endpoints/send-requests.md:55]: Changed the wording here since you last looked at this PR

**Pull request merged**